### DISCUSSION TITLE
Method to increase incorming HTTP request size

### DISCUSSION THREAD

**Discussion by csakthivel3773** [istio/istio#54124] [Q&A]
Method to increase incorming HTTP request size

Description:

I am getting the below error when the incoming request size is greater than 300 MB. The istio proxy sidecar container throws the below error and doesn't allow the request. How to increase the threshold value here.

ERROR:

503 UC upstream_reset_before_response_started{connection_termination}

**Comment by uttiyahazra** [comment]
@csakthivel3773 Is it evident that the above stated _503 UC upstream_reset_before_response_started{connection_termination_ issue is occurring due to limitation on incoming http request size and the issue is deterministically reproducible? Because in such case (shorter http request size) usually the error _431 Request Header Fields Too Large_ is observed, whereas the former one can occur due to variety of reasons including an issue with destination/upstream proxy server connection closing. Can you please apply an EnvoyFilter configuration which attempts a few retries on given status code 503 and check if the outcome remains same?
